backend/src/main/java/ca/bc/gov/hlth/mohums/controller/OrganizationsController.java

```diff
@@ -0,0 +1,65 @@
+package ca.bc.gov.hlth.mohums.controller;
+
+import ca.bc.gov.hlth.mohums.webclient.WebClientService;
+import net.minidev.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.*;
+
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.Map;
+
+
+@RestController
+public class OrganizationsController {
+
+    //todo: write the url, user and password somewhere else
+    @Value("${spring.datasource.url}")
+    private String url;
+    @Value("${spring.datasource.username}")
+    private String user;
+    @Value("${spring.datasource.password}")
+    private String password;
+    @Value("${spring.datasource.database}")
+    private String database;
+    @Value("${spring.datasource.organization_id}")
+    private String organization_id;
+    @Value("${spring.datasource.organization_name}")
+    private String organization_name;
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public OrganizationsController(WebClientService webClientService, @Value("${config.vanity-hostname}") String vanityHostname) {
+    }
+
+    public PreparedStatement connectAndPrepareStatement(String sql) throws SQLException {
+        return DriverManager.getConnection(url, user, password).prepareStatement(sql);
+    }
+
+    @GetMapping("/organizations")
+    public Collection<Object> getOrganizations() throws SQLException {
+        return jdbcTemplate.query("SELECT * FROM " + database + " ORDER BY " + organization_id,
+                (rs, rowNum) -> Map.of("id", rs.getString(organization_id),
+                        "name", rs.getString(organization_name)));
+    }
+
+    @GetMapping("/organizations/{organizationId}")
+    public Object getOrganization(@PathVariable String organizationId) throws SQLException {
+        return jdbcTemplate.queryForObject("SELECT * FROM " + database + " WHERE " + organization_id + " = ?", new Object[]{organizationId},
+                (rs, rowNum) -> Map.of("id", rs.getString(organization_id), "name", rs.getString(organization_name))
+        );
+    }
```

Comment:
Remove this resource and the corresponding frontend functionality. It's out of scope for now.

(Note regarding code review feedback, please feel free to ask questions and debate. It's difficult to give feedback without sounding bossy and arrogant, but that's not the intent. Some feedback will be important and objective, but lots will be minor and subjective.)

Reply:
noted! Thanks for the tip

Reply:
removed the resource